tf_helpers/lax_tests.py

```diff
@@ -0,0 +1,104 @@
+# Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+
+from tensorflow import nn
+import tensorflow as tf
+import lax
+from tensorflow.python.platform import test
+from absl.testing import parameterized
+import itertools
+import numpy as onp
+from tensorflow.python.ops import numpy_ops as tfnp
+from jax import numpy as jnp
+import jax
+import sys
+
+class TFConvGeneralTest(tf.test.TestCase, parameterized.TestCase):
+
+
```

Comment:
Nit: please use two empty lines between top level defs, and one between class methods (https://www.python.org/dev/peps/pep-0008/#blank-lines; here it's the opposite).